schemas/com.growthbook/experiment_viewed/jsonschema/1-0-0

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an GrowthBook experiment viewed event",
```

Comment:
Probably nice to link to https://docs.growthbook.io or similar so it's easier for people to get started if this isn't a native integration.

Reply:
Linked